modules/builds-use-custom-builder-image.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * assembly/builds
+// * builds/custom-builds-buildah.adoc
+
+
+[id='builds-use-custom-builder-image-{context}']
+= Use custom builder image
+
+You can define a BuildConfig that uses the custom strategy in conjunction with
+your custom builder image to execute your custom build logic.
+
+.Prerequisites
+
+* Define all the inputs that will go into creating your new custom builder image.
+* Build your custom builder image.
+
+.Procedure
+
+. Create a file named *buildconfig.yaml*. This file defines the BuildConfig that
+is created in your project and executed:
++
+[source,yaml]
+----
+kind: BuildConfig
+apiVersion: v1
+metadata:
+  name: sample-custom-build
+  labels:
+    name: sample-custom-build
+  annotations:
+    template.alpha.openshift.io/wait-for-ready: 'true'
+spec:
+  strategy:
+    type: Custom
+    customStrategy:
+      forcePull: true
+      from:
+        kind: ImageStreamTag
+        name: custom-builder-image:latest
+        namespace: <yourproject> <1>
+      exposeDockerSocket: true
```

Comment:
QE noted that this line should be removed.

Reply:
There are two other reference in the docs set for `DockerSocket`.  I'll open a separate PR to remove those. Thanks!